governance/polls/Adjust the Dust Parameter - November 29, 2021.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Increase the Dust Parameter - November 29, 2021
+summary: Signal your support or opposition to Increase the Dust Parameter from 10,000 DAI to 15,000 DAI for all vault-types excluding ETH-B and ETH-C.
+discussion_link: https://forum.makerdao.com/t/signal-request-adjusting-dust-parameter-2021-4/11598
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+   - Technical
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-29T16:00:00
+end_date: 2021-12-02T16:00:00
+---
+# Poll: Increase the Dust Parameter - November 29, 2021
```

Comment:
Could you change this to "Increase the Dust Parameter for Most Vault Types" like this poll:
https://github.com/mkrorbrkr/community/blob/mkrorbrkrpolls/governance/polls/Increase%20the%20Dust%20Parameter%20for%20Most%20Vault%20Types%20-%20March%2015%2C%202021.md